style.css

```diff
@@ -84,6 +116,204 @@ body {
   list-style: none;
 }
 
+.works-section {
+  background-color: inherit;
+  display: grid;
+  grid-template-columns: 100%;
+  padding: 7.12rem 1.5rem;
+  row-gap: 5.3rem;
+  justify-content: center;
+}
+
+.works-card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  border-radius: 1rem;
+}
+
+.works-card--img {
+  margin-bottom: 0.75rem;
+}
+
+.works-card--title {
+  font-weight: 700;
+  font-size: 2em;
+  color: #172b4d;
+  margin-bottom: 0.75rem;
+}
+
+.works-card--info {
+  display: flex;
+  justify-content: space-between;
+  padding: 0.5rem 0;
+  margin-right: 3.6rem;
+  margin-bottom: 0.75rem;
+}
+
+.works-card--info li {
+  list-style: none;
+  font-size: 0.8em;
+  font-weight: 600;
+  line-height: 1em;
+  color: #7a869a;
+}
+
+.works-card--info > li.canopy {
+  color: #344563;
+}
+
+.works-card--detail {
+  font-weight: 400;
+  font-size: 0.9em;
+  line-height: 24px;
+  color: #344563;
+  padding-right: 3rem;
+  margin-bottom: 0.75rem;
+}
+
+.works-card--languages {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 5.75rem;
+  margin-bottom: 0.75rem;
+  padding: 0;
+}
+
+.works-card--languages > li {
+  list-style: none;
+  padding: 0.25rem 0.75rem;
+  background-color: #ebebff;
+  border-radius: 0.5rem;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+}
+
+.works-card button {
+  align-self: start;
+}
+
+.about-myself-section {
+  display: grid;
+  row-gap: 1.5rem;
+  padding: 7.12rem 1.5rem;
+  border-top-right-radius: 100px;
+}
```

Comment:
- [ ] It is a great achievement to have a clear knowledge about `grid`. :+1: :1st_place_medal: . However, the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) require you to use flexbox to align the contents of `about section`. You should consider using `flexbox` instead for this particular section.